federation/pkg/federation-controller/namespace/namespace_controller.go

```diff
@@ -264,6 +276,9 @@ func (nc *NamespaceController) reconcileNamespace(namespace string) {
 		}
 
 		if !found {
+			nc.eventRecorder.Eventf(baseNamespace, api.EventTypeNormal, "UpdateCluster",
+				"Create namespace in cluster %s", cluster.Name)
```

Comment:
Its better to be explicit about is this Creat*ed* or Creat*ing*. I think it should be Creating here.